<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>869</width>
    <height>641</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QWidget" name="horizontalWidget" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="pushButton2">
         <property name="text">
          <string>Start</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton1">
         <property name="text">
          <string>Quit</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QStackedWidget" name="stackedWidget">
         <property name="currentIndex">
          <number>2</number>
         </property>
         <widget class="QWidget" name="page1">
          <layout class="QGridLayout" name="gridLayout_3">
           <item row="0" column="0">
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>This is page 1</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <layout class="QGridLayout" name="gridOfPlots">
             <item row="0" column="0">
              <widget class="PlotWidget" name="pw_grid_0" native="true"/>
             </item>
             <item row="0" column="1">
              <widget class="PlotWidget" name="pw_grid_1" native="true"/>
             </item>
             <item row="0" column="2">
              <widget class="PlotWidget" name="pw_grid_2" native="true"/>
             </item>
             <item row="1" column="0">
              <widget class="PlotWidget" name="pw_grid_3" native="true"/>
             </item>
             <item row="1" column="1">
              <widget class="PlotWidget" name="pw_grid_4" native="true"/>
             </item>
             <item row="1" column="2">
              <widget class="PlotWidget" name="pw_grid_5" native="true"/>
             </item>
             <item row="2" column="0">
              <widget class="PlotWidget" name="pw_grid_6" native="true"/>
             </item>
             <item row="2" column="1">
              <widget class="PlotWidget" name="pw_grid_7" native="true"/>
             </item>
             <item row="2" column="2">
              <widget class="QPlainTextEdit" name="outputText"/>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="page2">
          <layout class="QGridLayout" name="gridLayout_4">
           <item row="1" column="0">
            <layout class="QVBoxLayout" name="verticalLayout_6" stretch="1,5">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,0,0,0,0,0,0,0">
               <item>
                <widget class="PlotWidget" name="pw_chan_0" native="true"/>
               </item>
               <item>
                <widget class="PlotWidget" name="pw_chan_1" native="true"/>
               </item>
               <item>
                <widget class="PlotWidget" name="pw_chan_2" native="true"/>
               </item>
               <item>
                <widget class="PlotWidget" name="pw_chan_3" native="true"/>
               </item>
               <item>
                <widget class="PlotWidget" name="pw_chan_4" native="true"/>
               </item>
               <item>
                <widget class="PlotWidget" name="pw_chan_5" native="true"/>
               </item>
               <item>
                <widget class="PlotWidget" name="pw_chan_6" native="true"/>
               </item>
               <item>
                <widget class="PlotWidget" name="pw_chan_7" native="true"/>
               </item>
              </layout>
             </item>
             <item>
              <widget class="PlotWidget" name="pw_chan_main" native="true"/>
             </item>
            </layout>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>This is page 2</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="page3">
          <layout class="QGridLayout" name="gridLayout_6">
           <item row="0" column="0">
            <widget class="QWidget" name="widget" native="true">
             <layout class="QGridLayout" name="gridLayout_5">
              <item row="0" column="0">
               <layout class="QGridLayout" name="gridLayout">
                <item row="0" column="0">
                 <widget class="PlotWidget" name="pw_ampl_0" native="true"/>
                </item>
                <item row="0" column="2">
                 <widget class="PlotWidget" name="pw_ampl_2" native="true"/>
                </item>
                <item row="0" column="1">
                 <widget class="PlotWidget" name="pw_ampl_1" native="true"/>
                </item>
                <item row="1" column="0">
                 <widget class="PlotWidget" name="pw_ampl_3" native="true"/>
                </item>
                <item row="1" column="2">
                 <widget class="PlotWidget" name="pw_ampl_5" native="true"/>
                </item>
                <item row="1" column="1">
                 <widget class="PlotWidget" name="pw_ampl_4" native="true"/>
                </item>
                <item row="2" column="0">
                 <widget class="PlotWidget" name="pw_ampl_6" native="true"/>
                </item>
                <item row="2" column="1">
                 <widget class="PlotWidget" name="pw_ampl_7" native="true"/>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>869</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFIle">
    <property name="title">
     <string>FIle</string>
    </property>
    <addaction name="actionOpen"/>
   </widget>
   <widget class="QMenu" name="menuViews">
    <property name="title">
     <string>Views</string>
    </property>
    <addaction name="actionPage1"/>
    <addaction name="actionPage2"/>
   </widget>
   <addaction name="menuFIle"/>
   <addaction name="menuViews"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionPage1">
   <property name="text">
    <string>Page1</string>
   </property>
  </action>
  <action name="actionPage2">
   <property name="text">
    <string>Page2</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlotWidget</class>
   <extends>QWidget</extends>
   <header>pyqtgraph</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
